src/Conf.ml

```diff
@@ -118,7 +118,8 @@ let info =
          $(b,\\$HOME/.config) if $(b,\\$XDG_CONFIG_HOME) is undefined. The \
          global $(b,ocamlformat) file has the lowest priority, then the \
          closer the directory is to the processed file, the higher the \
-         priority."
+         priority. The global $(b,ocamlformat) file is only used when the \
```

Comment:
You could just say "It".

Reply:
I think it is clearer this way, sinceit is a long sentence and the previous "global ocamformat file" is really early in the sentence.